src/locale/pt-br/texts.po.json

```diff
@@ -1 +1 @@
"msgstr":["Enviar"]},"Invalid QR code":{"msgid":"Invalid QR code","msgstr":["QR code inválido"]},"OK":{"msgid":"OK","msgstr":["OK"]},"You don't have the requested token [${ tokenLabel }]":{"msgid":"You don't have the requested token [${ tokenLabel }]","msgstr":["Você não tem o token requisitado [${ tokenLabel }]"]},"Scan the QR code":{"msgid":"Scan the QR code","msgstr":["Leia o QR code"]},"You are connected to":{"msgid":"You are connected to","msgstr":["Você está conectado à"]},"Connected to":{"msgid":"Connected to","msgstr":["Conectado ao servidor"]},"Security":{"msgid":"Security","msgstr":["Segurança"]},"Create a new token":{"msgid":"Create a new token","msgstr":["Criar um novo token"]},"Register a token":{"msgid":"Register a token","msgstr":["Registrar um token"]},"About":{"msgid":"About","msgstr":["Sobre"]},"Unregister":{"msgid":"Unregister","msgstr":["Desregistrar"]},"UNREGISTER TOKEN":{"msgid":"UNREGISTER TOKEN","msgstr":["DESREGISTRAR TOKEN"]},"If you unregister this token **you won't be able to execute operations with it**, unless you register it again.":{"msgid":"If you unregister this token **you won't be able to execute operations with it**, unless you register it again.","msgstr":["Se você desregistrar esse token **você não conseguirá mais executar operações com ele**, a não ser que você o registre novamente."]},"You won't lose your tokens, they will just not appear on this wallet anymore.":{"msgid":"You won't lose your tokens, they will just not appear on this wallet anymore.","msgstr":["Você não irá perder os seus tokens, eles apenas não irão mais aparecer nessa wallet."]},"I want to unregister the token **${ tokenLabel }**":{"msgid":"I want to unregister the token **${ tokenLabel }**","msgstr":["Eu quero desregistrar o token **${ tokenLabel }**"]},"Unregister token":{"msgid":"Unregister token","msgstr":["Desregistrar token"]},"Copied to clipboard!":{"msgid":"Copied to clipboard!","msgstr":["Copiado!"]},"Create payment request":{"msgid":"Create payment request","msgstr":["Criar requisição de pagamento"]},"No internet connection":{"msgid":"No internet connection","msgstr":["Sem conexão com a internet"]},"Opening camera":{"msgid":"Opening camera","msgstr":["Abrindo a câmera"]},"Here is my address: ${ _this.state.address }":{"msgid":"Here is my address: ${ _this.state.address }","msgstr":["Esse é o meu endereço: ${ _this.state.address }"]},"New address":{"msgid":"New address","msgstr":["Novo endereço"]},"Share":{"msgid":"Share","msgstr":["Compartilhar"]},"Here is the configuration string of token ${ tokenLabel }: ${ configString }":{"msgid":"Here is the configuration string of token ${ tokenLabel }: ${ configString }","msgstr":["Essa é a configuração do seu token ${ tokenLabel }: ${ configString }"]},"Description":{"msgid":"Description","msgstr":["Descrição"]},"Date & Time":{"msgid":"Date & Time","msgstr":["Data & Hora"]},"ID":{"msgid":"ID","msgstr":["ID"]},"Public Explorer":{"msgid":"Public Explorer","msgstr":["Explorer Público"]}}}}
```

Comment:
If we are going to keep these, we should remove `src/locale/.gitignore`.

Reply:
Fixed.